### PR TITLE
Tweaks for sankeyNetwork

### PR DESCRIPTION
**Pull request by ijlyttle**
I had this lying around to address #15 - so I thought I would throw it over the wall as a PR.

There are a few things done for this PR:

1. Removes the font specification, to use the d3 default
2. Adjusts number of backslashes for the newline in the tooltip
3. Puts in a right arrow in the tooltip, between the source and the target
4. Adds an option for physical units to be displayed the tooltip.

### CODE REVIEW

**Comment by timelyportfolio** [comment]
@christophergandrud, I am planning to try to improve sankey a little.  Do you expect to incorporate this pull request?

**Comment by ijlyttle** [comment]
let me know what I can do to help - it may take me a little while to load this back into my brain, but I'll do what I can

**Comment by timelyportfolio** [comment]
Looks pretty clean to me, and I like your changes/improvements.  Will just wait to hear from @christophergandrud to see if anything needs to be resolved and if not an expected timeline for inclusion.

If I hear nothing, then I will start a `htmlwidget` dedicated to Sankey separate from `networkD3`.

**Comment by ijlyttle** [comment]
Sounds good to me.

File: R/sankeyNetwork.R [hunk 1]
```diff
@@ -39,7 +40,7 @@
 #' # Plot
 #' sankeyNetwork(Links = EngLinks, Nodes = EngNodes, Source = "source",
 #'              Target = "target", Value = "value", NodeID = "name",
-#               fontsize = 12, nodeWidth = 30)
+#               units = "TWh", fontsize = 12, nodeWidth = 30)
```
Comment (timelyportfolio) [inline R/sankeyNetwork.R]: believe `fontsize` should be `fontSize`

File: R/sankeyNetwork.R [hunk 2]
```diff
@@ -47,8 +48,8 @@
 #'
 #' @export
 
-sankeyNetwork <- function(Links, Nodes, Source, Target, Value, NodeID,
-    height = NULL, width = NULL, colourScale = "d3.scale.category20()",
+sankeyNetwork <- function(Links, Nodes, Source, Target, Value, NodeID, 
+    units = "", height = NULL, width = NULL, colourScale = "d3.scale.category20()",
     fontsize = 7, nodeWidth = 15, nodePadding = 10)
```
Comment (timelyportfolio) [inline R/sankeyNetwork.R]: believe `fontsize` should be `fontSize`

File: R/sankeyNetwork.R [hunk 3]
```diff
@@ -75,7 +76,8 @@ sankeyNetwork <- function(Links, Nodes, Source, Target, Value, NodeID,
         colourScale = colourScale,
         fontsize = fontsize,
```
Comment (timelyportfolio) [inline R/sankeyNetwork.R]: believe `fontsize` should be `fontSize`

File: inst/htmlwidgets/sankeyNetwork.js [hunk 4]
```diff
@@ -118,7 +119,7 @@ HTMLWidgets.widget({
             .attr("text-anchor", "end")
             .attr("transform", null)
             .text(function(d) { return d.name; })
-            .style("font", options.fontsize + "px serif")
+            .style("font", options.fontsize + "px")
```
Comment (timelyportfolio) [inline inst/htmlwidgets/sankeyNetwork.js]: believe `fontsize` should be `fontSize`

File: R/sankeyNetwork.R (hunk 3 above)
Comment (timelyportfolio) [inline R/sankeyNetwork.R]: to be consistent with other functions, probably should add a `fontFamily` argument.

File: inst/htmlwidgets/sankeyNetwork.js (hunk 4 above)
Comment (timelyportfolio) [inline inst/htmlwidgets/sankeyNetwork.js]: `fontFamily` if provided ? see previous line note

Comment (timelyportfolio) [inline inst/htmlwidgets/sankeyNetwork.js]: `.style("font", options.fontsize + "px")`

should be

`.style("font-size", options.fontSize + "px")`

**Comment by ijlyttle** [comment]
I can make the change `fontsize -> fontSize` pretty quickly, and push.

I'll need to poke around a bit to see how to incorporate `fontFamily`

**Comment by timelyportfolio** [comment]
@ijlyttle I implemented this way which appears consistent with the other functions

[r lines](https://github.com/timelyportfolio/networkD3/blob/improve/sankey/R/sankeyNetwork.R#L52)

[js lines](https://github.com/timelyportfolio/networkD3/blob/improve/sankey/inst/htmlwidgets/sankeyNetwork.js#L122-L123)

**Comment by ijlyttle** [comment]
I have pushed to my branch - should I instead make a pull request to https://github.com/timelyportfolio/networkD3/tree/improve/sankey?

**Comment by timelyportfolio** [comment]
Thanks for making those changes.  Let's wait to hear from @christophergandrud.

**Pull request merged**